build.sbt

```diff
@@ -67,11 +67,23 @@ ThisBuild / developers := List(
 
 ThisBuild / githubWorkflowJavaVersions := Seq("11", "17").map(JavaSpec.temurin(_))
 ThisBuild / tlCiReleaseBranches := Seq("series/0.7")
-ThisBuild / tlSitePublishBranch := Some("series/0.7")
+// ThisBuild / tlSitePublishBranch := Some("series/0.7")
+ThisBuild / tlSitePublishBranch := Some("doc/versioned-0.7")
```

Comment:
This needs to be reverted before merge, right?

Reply:
Yup, that was just to get it to deploy on my fork.